### PR TITLE
count others' kills against you on the deathmatch score board

### PR DESCRIPTION
**Pull request by fabiangreffrath**
much more interesting with more than 2 players

### CODE REVIEW

**Comment by fabiangreffrath** [comment]
Does anybody know an interesting 4 player deathmatch demo?

File: src/st_widgets.c [hunk 1]
```diff
@@ -764,7 +764,7 @@ static void UpdateMonSec(sbe_widget_t *widget)
 
         for (int i = 0; i < MAXPLAYERS; ++i)
         {
-            int result = 0;
+            int result = 0, others = 0;
```
Comment (fabiangreffrath) [inline src/st_widgets.c:767]: This has become larger than the actual monsec widget code. Should I factor it out into a separate function?

**Comment by rfomin** [comment]
> Does anybody know an interesting 4 player deathmatch demo?

I only know coop demos from DSDA. I don't think anyone plays deathmatch using Woof. BTW, the KEX port has decent multiplayer (not demo compatible).

**Comment by fabiangreffrath** [comment]
This has a link to a deathmatch demo:

https://github.com/fabiangreffrath/crispy-doom/pull/921

**Comment by fabiangreffrath** [comment]
> This has a link to a deathmatch demo:

Confirmed, the numbers match the rows and columns on the intermission screen.

**Review by rfomin — APPROVED** [review]

**Pull request merged**